### ISSUE TITLE
module 'cryodrgn.dataset' has no attribute 'load_particles'

### ISSUE THREAD

**Issue by finde159** [ml-struct-bio/cryodrgn#296]
Hi,

I've recently been using cryoDRGN and was trying to run "cryoDRGN_filtering_template.ipynb", but it raises an error saying "AttributeError: module 'cryodrgn.dataset' has no attribute 'load_particles'" in the 16th box.
I have looked at the codes of dataset.ipynb in cryodrgn module and I couldn't find a function or anything named 'load_particles'.
Other modules and functions seem to be imported appropriately, but the line where it says:
particles = dataset.load_particles(config...)
keeps on raising an error.

Please let me know if there is any solution for this.

Thanks.
Mark.

**Comment by zhonge** [comment]
Hi Mark, Can you try the latest released version 2.3.0 instead of latest development version? You can install the latest release version with either:

```
$ pip install cryodrgn
```

or if you are installing from the source code:
```
$ git checkout 2.3.0
$ pip install .
```



Thanks for reporting this issue. We are currently refactoring the data loading, and it looks like we need to update some of the functions in the jupyter notebooks.

**Comment by michal-g** [comment]
We've now updated the filtering notebook in v3.0.1-b to avoid conflicts with the latest development version.
